### PR TITLE
Introduced standard label format for EAN13 layout

### PR DESCRIPTION
**Pull request by danigoffi**
Introduced standard label format for EAN13 layout. The boolean parameter "StandardizeLabel" is used to switch between original and standardized label (default is true).

### CODE REVIEW

**Comment by systematic83** [comment]
Not yet perfect I know... but it's a starting point.
I'm confident that someone can improve this in the future.

File: BarcodeLib/BarcodeLib.cs [hunk 1]
```diff
@@ -13,7 +13,8 @@
  *                 Barcode Library                   *
  *                                                   *
  *             Written by: Brad Barnhill             *
- *              Started on: 09-21-2007               *
+ *                   Date: 09-21-2007                *
```
Comment (barnhill) [inline BarcodeLib/BarcodeLib.cs]: youll want to rebase for this ... as I changed this

File: BarcodeLib/BarcodeLib.cs [hunk 2]
```diff
@@ -567,6 +578,172 @@ private Bitmap Generate_Image()
 
                         break;
                     }//case
+
```
Comment (barnhill) [inline BarcodeLib/BarcodeLib.cs]: there is a LOT of excess whitespace in here

Comment (danigoffi) [reply BarcodeLib/BarcodeLib.cs]: Sorry, my fault. I'm a C# newbie and I'm trying to learn by studying brilliant projects like yours. Sometimes I use whitespaces to keep my code tidy.. I'm pretty sure that you will find some others silly mistakes in my code, so please be patient with me and thanks for helping me learn!

Comment (barnhill) [reply BarcodeLib/BarcodeLib.cs]: No need for apologies as I was new at one point in time :)

I should have the build fixed if you want to rebase it should pass the build and Ill get around to checking out the functionality of it here shortly.  Got super busy last night and couldnt do more than fix the build.

**Comment by barnhill** [comment]
what do you think of this label ... its most of your work with some changes to make it dependent on the bar width and adjustment of the start boxes since they seemed to be off slightly
ean13-label

**Comment by barnhill** [comment]
If you like the way this looks Ill create a branch in my repo and put a PR out and have you review it

**Comment by danigoffi** [comment]
I think that is a very good result! Do you think that will be the chance to have the first digit as the same size of all the others? I know that it's not easy because of the variable size of the margins and that's why I have forced the right alignment when using the standardized label.. however I think that is a very good result, the white background rectangles looks much better now.

**Comment by barnhill** [comment]
@danigoffi the reason I shrank the first digit was that I saw in a lot of EAN-13 barcodes online that they always seems to shrink that first digit

**Comment by danigoffi** [comment]
@barnhill I have asked about the first character based on what I have read in the EAN-13 page in Wikipedia (https://en.wikipedia.org/wiki/International_Article_Number) but I don't think that it could make a big difference.. is a very good result however

**Comment by barnhill** [comment]
sorry been running into some issues

problem

**Comment by barnhill** [comment]
@danigoffi sorry its been a while but I got around to looking at this tonight and here is the result:

ean-label

**Comment by barnhill** [comment]
Closing this as it is fixed by bd7bf7940ff67f1aef36fb89eb226d58656f424f

**Comment by barnhill** [comment]
@danigoffi  THANK YOU SO MUCH for the pull request.  I like seeing people enthusiastic about adding to this lib.

**Comment by danigoffi** [comment]
@barnhill thanks to people like you that makes this stunning library available to everyone!

**Pull request closed without merging**